test/workbox-background-sync/node/lib/test-Queue.mjs

```diff
@@ -376,7 +382,13 @@ describe(`[workbox-background-sync] Queue`, function() {
 
       await queue.addRequest(new Request('/three'));
       await queue.addRequest(new Request('/four'));
-      await queue.replayRequests();
+      try {
+        await queue.replayRequests();
```

Comment:
Please use expectError here and make sure the error is what you expect.

Reply:
Done